cmd/esc/cli/testdata/env-tag-mv.yaml

```diff
@@ -0,0 +1,24 @@
+run: |
+  esc env tag ls test-org/env
+  esc env tag mv test-org/env team esc
+  esc env tag mv test-org/env team owner pulumi
```

Comment:
what does this do? does this rename the tag and give it a new value?

Reply:
👍